common/date.go

```diff
@@ -0,0 +1,67 @@
+package common
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// ParseDate parses a date string of the format YYYY-MM-DD
+func ParseDate(date string) (t time.Time, err error) {
```

Comment:
I had a bunch of trouble getting this to work... really confusing with the month day being represented by the 02 and 01 respectively. BUT THANKS. this makes sense... Do we still want to include this library for date range parsing? Also nice to just have the layout predefined here I think... I'll update the code